run/default.nix

```diff
@@ -1,4 +1,5 @@
-{ lib
+{ pkgs
```

Comment:
I  typically tend to put the specific packages here instead of the top level pkgs. So something like

```suggestion
{ makeWrapper
, qemu
```

I don't feel strongly about this but would prefer consistency.